test/controllers/goal_controller_test.exs

```diff
@@ -0,0 +1,97 @@
+defmodule Healthlocker.GoalControllerTest do
+  use Healthlocker.ConnCase
+
+  alias Healthlocker.Post
+  alias Healthlocker.User
+
+  setup do
+    %User{
+      id: 123456,
+      name: "MyName",
+      email: "abc@gmail.com",
+      password_hash: Comeonin.Bcrypt.hashpwsalt("password")
+    } |> Repo.insert
+
+    {:ok, user: Repo.get(User, 123456) }
+  end
+
+  @valid_attrs %{content: "some content"}
+  @invalid_attrs %{content: ""}
+
+  test "renders index.html on /goal", %{conn: conn, user: user} do
+    conn = build_conn()
+        |> assign(:current_user, user)
```

Comment:
@katbow please see: http://blog.simonstrom.xyz/elixir-testing-tag-tests-in-exunit
(_for a much simpler way of authenticating in tests..._) 👍